config/internal/apiserver/default/kfp_launcher_config.yaml.tmpl

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
 data:
+  {{ if .APIServer.CustomKfpLauncherConfig }}
+  providers: |
```

Comment:
(Context: "pipeline root" is a terrible name, and what it actually means is "the base folder in my object storage where all my stuff will go.")

Hm, looks like the user can only set the fields under `providers` -- i.e., they can't change `defaultPipelineRoot`. That seems incorrect to me.

1. If I'm allowed to override this ConfigMap with my own for the purposes of making the object storage connection exactly what I want, I should absolutely be able to have my pipeline root (base object storage path) set right along side in that same override ConfigMap.

2. Given the (original -- I suggested a change) desription of the field above, as a user, I would be caught off guard if only a subset of the ConfigMap was replaced. I'm expecting to be able to control everything under data.